APE14.rst

```diff
@@ -182,6 +190,9 @@ low-level API recommends:
             data. This can be used for example if implementing slicing of WCS
             objects. This is an optional property, and it should return `None`
             if a shape is not known or relevant.
```

Comment:
Shouldn't _"...if a shape is not known or relevant."_ be something like _"... if a shape is neither known nor relevant."_? The way this is currently written is confusing to me (but it just could be my English): does **not** apply to **known** and not to **relevant** or does **not** apply to _both_ **known** _and_ **relevant**?

Same goes for `numpy_shape`, `pixel_shape`.